### PR TITLE
feature/new referral component

### PR DESCRIPTION
**Pull request by emilyjablonski**
See this issue for more information: #1142

For some reason I couldn't get the phone icon to render within the story, but it renders just fine within the application as seen in the screenshot below:
Screen Shot 2021-04-20 at 2 02 02 PM

### CODE REVIEW

**Comment by emilyjablonski** [comment]
@jaredcwhite Hoping to get your review

File: ui-components/src/page_components/listing/listing_sidebar/ReferralApplication.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  description: string
+  phoneNumber: string
+  title: string
+}
+
+const ReferralApplication = (props: ReferralApplicationProps) => {
+  const linkedPhoneNumber = `tel:${props.phoneNumber.replace(/[-()]/g, "")}`
+
+  return (
+    <section className="aside-block -mx-4 pt-0 md:mx-0 md:pt-4">
```
Comment (jaredcwhite) [inline ui-components/src/page_components/listing/listing_sidebar/ReferralApplication.tsx:15]: We can remove all the utility classes now and just use `aside-block` (due to merge of #1140)

**Review by jaredcwhite — CHANGES_REQUESTED** [review]
Just one comment due to recent master styling update, otherwise LGTM!

**Comment by jaredcwhite** [comment]
LGTM!

**Pull request merged**